src/pages/julieLee/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./Login.scss";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+
+function LoginJulie() {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate("/main-julie");
+  };
+
+  const [identify, setIdentify] = useState("");
+  const handleIdInput = (event) => setIdentify(event.target.value);
+  const [password, setPassword] = useState("");
+  const handlePwInput = (event) => setPassword(event.target.value);
```

Comment:
```suggestion
  const [ inputs, setInputs ] = useState({
    identify: "",
    password: "",
  });
  const handleInputByName = (event) => {
    setInputs(prevInputs => {
      ...prevInputs,
      [event.target.name]: event.target.value
    });
  }
```

input이 많아진다면 이렇게도 관리해보세요~